pkg/datapath/loader/hash.go

```diff
@@ -14,10 +14,10 @@ import (
 )
 
 var (
-	// DatapathSHA256 is set during build to the SHA across all datapath BPF
+	// datapathSHA256 is set during build to the SHA across all datapath BPF
 	// code. See the definition of CILIUM_DATAPATH_SHA256 in Makefile.defs for
 	// details.
-	DatapathSHA256 string
+	datapathSHA256 string
```

Comment:
Doesn't the makefile need updating if we change the name?

Reply:
Good catch!